### ISSUE TITLE
release (or catch) based on movesets

### ISSUE THREAD

**Issue by kenkclam** [PokemonGoF/PokemonGo-Bot#4970]
### Short Description
release pokemon based on their movesets

Please let me know where to find it if the feature is already available 

### Possible solution
maybe:
"Electabuzz": {"release_below_moveset1_DPS": 8, "release_below_moveset2_DPS": 21, "logic": "or"},

### How it would help others
release pokemon based on the DPS damage per second of their move sets. And filter move set 1 and move set 2 separately.
Move set is definitely more important than cp for me

**Comment by RedViper9** [comment]
Have you tried the Optimizer?
https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/docs/pokemon_optimizer.md

**Comment by kenkclam** [comment]
Yes, but I cannot figure out how to keep all the pokemon with a certain dps, instead of sorting and keep the best n number of them. I also cannot figure out a way to filter quick move and charge move separately.
There is this parameter "attack_perfection", which I don't understand what it is doing, maybe the Optimiser can do what I want, I just cannot figure it out now.

**Comment by avexus** [comment]
This relates to my request of nicknaming movesets according to dps. Apparently movesets are at least as important as the IV value. So, if we have a way to nickname properly, we can manually put a favorite sign to the pokemon and they never gets transferred. Or do it automatically.

**Comment by kenkclam** [comment]
Yes, it is related to, but still quite different from the nicknaming request.

I don't want the bot to nickname my pokemon, it makes it too obvious that I used a bot when I put the pokemon in gyms. And I don't want to manually transfer the pokemon according to the nickname, I want the bot to do it automatically.

**Comment by Anakin5** [comment]
See https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/docs/pokemon_optimizer.md#keep-sort, espcially dps1 and dps2

It is not implemented like what you want because it is hard to see why you would want to keep a Pokemon with dps 8 if you have 50 of the same Pokemon with dps 9. That is why we say "keep the X best".
